cloudformation/aws-parallelcluster.cfn.json

```diff
@@ -1421,6 +1448,54 @@
         "MessageRetentionPeriod": 1209600
       }
     },
+    "FSXSubstack": {
+      "Type": "AWS::CloudFormation::Stack",
+      "Properties": {
+        "Parameters": {
+          "FSXOptions": {
+            "Ref": "FSXOptions"
+          },
+          "ComputeSecurityGroup": {
+            "Fn::If": [
+              "CreateSecurityGroups",
+              {
+                "Ref": "ComputeSecurityGroup"
+              },
+              {
+                "Ref": "VPCSecurityGroupId"
+              }
+            ]
+          },
+          "SubnetId": {
+            "Ref": "MasterSubnetId"
+          }
+        },
+        "TemplateURL": {
+          "Fn::Sub": [
+            "https://${s3_domain}/seaam-pcluster/templates/fsx-substack-${version}.cfn.json",
```

Comment:
Needs to be changed

Reply:
will you change it before committing this file?